### PR TITLE
Updated to 2021R2 UnitTests and Examples

### PR DESCRIPTION
**Pull request by maxcapodi78**
Added logger to aedt_exception_handler

### CODE REVIEW

File: pyaedt/siwave.py [hunk 1]
```diff
@@ -46,7 +46,6 @@ class Siwave:
     """
 
     @property
-    @aedt_exception_handler
```
Comment (MaxJPRey) [inline pyaedt/siwave.py:49]: @maxcapodi78 Is it expected to remove the decorator `@aedt_exception_handler` for siwave?

**Review by MaxJPRey — APPROVED** [review]

File: examples/00-basic-examples/03_5G_antenna_example.py [hunk 2]
```diff
@@ -3,12 +3,12 @@
 --------------------------------------
 This example shows how to use HFSS 3D Layout to create and solve a 5G linear array antenna.
 """
-# sphinx_gallery_thumbnail_path = 'Resources/3dlayout.png'
+# sphinx_gallery_thumbnail_path = 'Resources/5gantenna.png'
```
Comment (MaxJPRey) [inline examples/00-basic-examples/03_5G_antenna_example.py:6]: @maxcapodi78 Good catch!

File: pyaedt/hfss.py [hunk 3]
```diff
@@ -110,13 +110,13 @@ class Hfss(FieldAnalysis3D, object):
...
-            )
-        except:
-            return "HFSS Module"
+    # def __repr__(self):
+    #     try:
+    #         return "HFSS {} {}. ProjectName:{} DesignName:{} ".format(
+    #             self._aedt_version, self.solution_type, self.project_name, self.design_name
+    #         )
+    #     except:
+    #         return "HFSS Module"
```
Comment (MaxJPRey) [inline pyaedt/hfss.py:119]: @maxcapodi78 Do you think that we can get rid of the __repr__ method?

File: pyaedt/siwave.py (hunk 1 above)
Comment (maxcapodi78) [reply pyaedt/siwave.py:49]: I've removed from some properties where it's not needed

File: pyaedt/hfss.py (hunk 3 above)
Comment (maxcapodi78) [reply pyaedt/hfss.py:119]: It's actually making the code explode under certain conditions.

**Pull request merged**